apps/contributor/tests/contributor_team_test.py

```diff
@@ -0,0 +1,36 @@
+import typing
+
+import pytest  # type: ignore[reportMissingImports]
+from django.core.exceptions import ValidationError
+
+from apps.contributor.factories import ContributorTeamFactory, ContributorUserFactory
+from apps.user.factories import UserFactory
+from main.tests import TestCase
+
+
+class TestContributorTeam(TestCase):
+    @typing.override
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = UserFactory.create()
+        cls.user_resource_kwargs = dict(
+            created_by=cls.user,
+            modified_by=cls.user,
+        )
+        cls.contributor_team = ContributorTeamFactory.create(**cls.user_resource_kwargs)
+        cls.contributor_user = ContributorUserFactory.create(
+            user_id="test_id",
+            team=cls.contributor_team,
+        )
+
+    def test_cannot_archive_team_with_members(self):
+        self.contributor_team.is_archived = True
+        with pytest.raises(ValidationError):
+            self.contributor_team.save()
```

Comment:
Let's try this one

```suggestion
        with self.assertRaises(ValidationError):
            self.contributor_team.save()
```
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

Reply:
@susilnem This changes creates precommit issues.

Reply:
@thenav56 @susilnem @Rup-Narayan-Rajbanshi Let's discuss about this issue.

Reply:
recommended by : https://docs.astral.sh/ruff/rules/pytest-unittest-raises-assertion/